charts/opensearch/values.yaml

```diff
@@ -279,7 +279,7 @@ enableServiceLinks: true
 protocol: https
 httpPort: 9200
 transportPort: 9300
-metricsPort: 9600
+metricsPort: 9200
```

Comment:
The metrics port here is for Performance Analyzer which utilizes the 9600 port
https://github.com/opensearch-project/helm-charts/commit/69daf9764e59088c71c263d80cf25093a65a5785

Reply:
I wonder if we need to add a new var for other metrics?

Reply:
@eyenx if we want to use this `metricsPort` (`The metrics port (for Performance Analyzer) that Kubernetes will use for the service`) then it should be 9600 right ?

Reply:
Does the performance Analazer actually expose metrics in prometheus format? If not I would still go for this, ut add 9600 as separate port. How can I enale the performance Analyzer plugin?

Reply:
So I enabled the plugin following the guide, but I still see no metrics when I try to curl the Port 

```
opensearch-cluster-master-0:/$ curl localhost:9600/_plugins/_performanceanalyzer/metrics  -vvv
06:45:58.738363 [0-x] == Info: [READ] client_reset, clear readers
06:45:58.738550 [0-0] == Info: Host localhost:9600 was resolved.
06:45:58.738685 [0-0] == Info: IPv6: ::1
06:45:58.738793 [0-0] == Info: IPv4: 127.0.0.1
06:45:58.738918 [0-0] == Info: [SETUP] added
06:45:58.739051 [0-0] == Info:   Trying [::1]:9600...
06:45:58.939510 [0-0] == Info:   Trying 127.0.0.1:9600...
```


```
opensearch-cluster-master-0:/$ curl -k https://localhost:9200/_plugins/_performanceanalyzer/config -u user:pw
{"performanceAnalyzerEnabled":true,"rcaEnabled":false,"loggingEnabled":false,"shardsPerCollection":0,"batchMetricsEnabled":false,"threadContentionMonitoringEnabled":false,"batchMetricsRetentionPeriodMinutes":7}
```

Reply:
I think when you enable PA you would need to query through 9600
https://opensearch.org/docs/latest/monitoring-your-cluster/pa/index/#example-api-query-and-response

Reply:
As mentioned above, I tried curling 9600, but no response. So this is why I went for 9200 at the end. Did anyone ever got this working?

Reply:
You would need to separately start the PA process I think.
Not an expert on that tho.

Reply:
Please another review, but waiting for #594 before bumping the version here. @peterzhuamazon @prudhvigodithi